roles/tas_single_node/defaults/main.yml

```diff
@@ -7,6 +7,20 @@ tas_single_node_system_packages:
 
 tas_single_node_podman_network: rhtas
 
+tas_single_node_rekor_templates:
```

Comment:
I don't think we want customers to modify these, they serve more as constants rather than configuration values. Let's move them to vars.

Reply:
Noted will make the changes